gen/testdata/thrift/collision.thrift

```diff
@@ -50,3 +50,13 @@ union union_collision {
 struct WithDefault {
 	1: required struct_collision pouet = struct_constant
 }
+
+struct AccessorNoConflict {
+    1: optional string getname
+    2: optional string get_name
+}
+
+struct AccessorConflict {
+    1: optional string name
+    2: optional string get_name (go.name = "GetName2")
```

Comment:
This is potentially a breaking change, right?

Reply:
Yeah, this is potentially breaking but for unlikely conflicts, we've taken the policy of suggesting "use `go.name` in case of conflict" rather than cutting a 2.0.

From a quick look at all IDLs in our system, I don't see anything this would actually cause conflicts for. When people use "getFoo" in the request struct, "foo" is usually an optional field in the response struct.